File: samples/server/petstore/java-helidon-server/se/src/main/java/org/openapitools/server/model/SingleRefType.java
package org.openapitools.server.model;



import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonValue;

/**
 * Gets or Sets SingleRefType
 */

public enum SingleRefType {

    ADMIN("admin"),
    USER("user");

    private String value;

    SingleRefType(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return value;
    }

    @Override
    public String toString() {
        return String.valueOf(value);
    }

    @JsonCreator
    public static SingleRefType fromValue(String text) {
        for (SingleRefType b : SingleRefType.values()) {
            if (String.valueOf(b.value).equals(text)) {
                return b;
            }
        }
        throw new IllegalArgumentException("Unexpected value '" + text + "'");
    }
}

File: samples/server/petstore/php-laravel/lib/app/Models/SingleRefType.php
<?php
/**
 * SingleRefType
 */
namespace app\Models;

/**
 * SingleRefType
 */
class SingleRefType
{
    /**
     * Possible values of this enum
     */
    const ADMIN = 'admin';

    const USER = 'user';

    /**
     * Gets allowable values of the enum
     * @return string[]
     */
    public static function getAllowableEnumValues()
    {
        return [
            self::ADMIN,
            self::USER
        ];
    }
}
